modules/editor/editor.js

```diff
@@ -1,5 +1,6 @@
 export default class Editor {
-	constructor( id, settings ) {
-		document.getElementById( id ).innerHTML = settings.content;
+	constructor( id ) {
```

Comment:
I find this breaks the data flow we're trying to build `postcontent => state => render`